addons/resource.language.en_gb/resources/strings.po

```diff
@@ -2997,12 +2997,12 @@ msgstr ""
 
 #: system/settings/settings.xml
 msgctxt "#641"
-msgid "Files with ReplayGain information"
+msgid "PreAmp for files with ReplayGain information"
```

Comment:
Is there  a better word for "PreAmp", which sounds rather technical, or is this something somebody familiar with "ReplayGain" immediately should understand, because it is well-known terminology?

Reply:
different softwares uses "Pre-amp" or "Preamp" or also "Pre-AMP"
but if we want avoid this term a generic idea could be
`Adjustment for files with ReplayGain information`

Reply:
Preamp is a common concept in ReplayGain, and most ReplayGain-compatible players implement a preamp setting. See the [ReplayGain standard player requirements](https://wiki.hydrogenaud.io/index.php?title=ReplayGain_2.0_specification#Pre-amplification).